nova/compute/manager.py

```diff
@@ -1822,7 +1822,6 @@ def _do_build_and_run_instance(self, context, instance, image,
             request_spec, filter_properties, admin_password, injected_files,
             requested_networks, security_groups, block_device_mapping,
             node=None, limits=None, host_list=None):
-
```

Comment:
Could you remove also the white-space changes, like this one?

Reply:
Sure sorry about that.